src/components/Checkbox/Checkbox.module.scss

```diff
@@ -0,0 +1,103 @@
+@import '../../styles/colors';
+
+.ZuiCheckbox {
```

Comment:
nit: `.Checkbox` - doesn't need `zUI` at least for now

Reply:
updated the class name